modules/test-generated-policies-ng20.adoc

```diff
@@ -5,7 +5,7 @@
 [id="test-generated-policies-ng20_{context}"]
 = Testing generated policies in the network graph
 
-After you download the network policies that {product-title-short} generates, you can test them by applying them to your cluster.
+After you download the network policies that {product-title-short} generates, you can test them by applying them to your cluster by using the CLI or by using your automated deployment procedures. You cannot apply generated network policies directly in the network graph.
```

Comment:
Suggestion:

```suggestion
After you download the network policies that {product-title-short} generates, you can test them by applying them to your cluster through the CLI or your automated deployment procedures. You cannot apply generated network policies directly in the network graph.
```

Reply:
I think I should keep "by using" per IBM Style (Through - Use with caution - Do not use to mean "by using".)